lang/cs/Org.Apache.REEF.IMRU/OnREEF/Parameters/EnforceCloseTimeout.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+// 
+//   http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+using Org.Apache.REEF.Tang.Annotations;
+
+namespace Org.Apache.REEF.IMRU.OnREEF.Parameters
+{
+    /// <summary>
+    /// When driver sends clsoe event to a task, it would expect the task to clsoe gracefully. 
+    /// After specified time out, if the task is still not closed, the close handler will throw exception enforcing the task to close. 
+    /// </summary>
+    [NamedParameter("Enfore the task to close time out in millisecond", "EnforceCloseTimeout", "1000")]
+    internal sealed class EnforceCloseTimeout : Name<int>
```

Comment:
IMRU task configuration is set in IMRUDriver that is in IMRU project. If we would like client to set it, IMRU client is also in IMRU project.